library/src/main/java/com/almeros/android/multitouch/RotateGestureDetector.java

```diff
@@ -60,6 +60,8 @@ public void onRotateEnd(RotateGestureDetector detector) {
     
     private final OnRotateGestureListener mListener;
     private boolean mSloppyGesture;
+    private float mFocusX;
```

Comment:
This looks good, but would you mind moving this up to the `TwoFingerGestureDetector`, as it is specific to multi touch gestures, not rotate gestures?